### PR TITLE
Issue#10 link to product page

### PR DESCRIPTION
**Pull request by Gautam2305**
- made links for product listing page
- some minor style and design changes

deploy site: https://incandescent-madeleine-d61986.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,44 +29,44 @@ <h6> Profile</h6></a>
...
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-5.png" alt=""/></a>
         </div>
         <h5 class="brand-heading"> TOP-BRANDS</h5>
         <div class="top-brand-container">
-            <img src="/assets/sale-pic-6.png" alt="">
-            <img src="/assets/sale-pic-7.png" alt="">
-            <img src="/assets/sale-pic-8.png" alt="">
-            <img src="/assets/sale-pic-9.png" alt="">
-            <img src="/assets/sale-pic-10.png" alt="">
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-6.png" alt=""/></a>
```
Comment (shubhamkumar648) [inline index.html:44]: you can write some name alt attributes

**Review by shubhamkumar648 — APPROVED** [review]
The home of UI looks neat and clean good use of folder structure , good to go

File: index.html (hunk 1 above)
Comment (Gautam2305) [reply index.html:44]: But they are images which have to change on sales everytime..hence i left it like that only....Thank you for your suggestion ❤️

File: pages/wishlist_page/wishlist.html [hunk 2]
```diff
@@ -92,25 +92,6 @@ <h6> Profile</h6></a>
...
-        </div>
-        <div class="card-container">
-            <div class="prod-img-container">
-                <div class="prod-img">
-                    <img src="/assets/product-img.png" alt="sweatshirt">
-                    <div class="wishlist-btn"><i class="fa-solid fa-heart"></i></div>
-                </div>
-            </div>
-            
             <div class="prod-name"><h4>PUMA </h4><span><p> Men Colourblocked Sweatshirt</p></span></div>
```
Comment (coldpigli) [inline pages/wishlist_page/wishlist.html:95]: would suggest using sematic tags instead of divs

**Review by coldpigli — APPROVED** [review]
Would suggest using semantic tags. Everything else looks good.

**Pull request merged**